Label i915 sysfs files properly

ceiba1985 opened the pull request:
Label i915 sysfs files properly on MBL board, so surfaceflinger, bootanim, system_server etc. can access the related files.

Test done: Android 14 on MBL board can boot successfully.

Tracked-On: OAM-118227

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

ceiba1985 committed:
Label i915 sysfs files properly

Label i915 sysfs files properly on MBL board, so surfaceflinger,
bootanim, system_server etc. can access the related files.

Test done: Android 14 on MBL board can boot successfully.

Tracked-On: OAM-118227
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon-merge/455
